ml-agents/mlagents/trainers/agent_processor.py

```diff
@@ -258,3 +258,8 @@ def __init__(
             self.behavior_id
         )
         self.publish_trajectory_queue(self.trajectory_queue)
+
+    def set_environment_stats(self, env_stats: Dict[str, float]) -> None:
+        for stat_name, val in env_stats.items():
+            # full_stat = f"Environment/{stat_name}"
```

Comment:
This looks like dead code, if it is not, maybe add an extra comment on this line about in which situations this should be considered.

Reply:
Done